easybuild/easyconfigs/b/BatMeth2/BatMeth2-2.1_fix-automake-rerun.patch

```diff
@@ -0,0 +1,26 @@
+diff -ru BatMeth2-BatMeth2-v2.1.orig/Makefile.in BatMeth2-BatMeth2-v2.1/Makefile.in
```

Comment:
@Darkless012 Short patch description + author is missing (don't forget to fix SHA256 checksum as well)